roles/kubernetes-tools/tasks/_suse.yml

```diff
@@ -4,7 +4,7 @@
     name: vm
     tasks_from: install_suse_package
   vars:
-    suse_packages: [git, python310, python310-pip, python310-devel, jq, curl, wget, nmap, lsof]
+    suse_packages: [git, python310, python310-pip, python310-devel, jq, curl, wget, nmap, lsof, python310-dbm]
```

Comment:
As previously mentioned, I believe this should only be included in the `ironicclient` role. `kubernetes-tools` is executed for both `network` and `core` VMs and I don't see this package being useful in the `network` one.